### PR TITLE
Added scroll to top

### PR DESCRIPTION
**Pull request by rishsingh07**
Added scroll to top

### CODE REVIEW

File: src/features/common/Layout/index.tsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import CookiePolicy from './CookiePolicy';
 import Header from './Header';
 import Navbar from './Navbar';
 import RedeemPopup from './RedeemPopup';
-
+import Scroll from './BackToTop/backToTop'
+import BackToTop from './BackToTop/backToTop';
```
Comment (norbertschuler) [inline src/features/common/Layout/index.tsx:9]: This looks like an unnecessary redundant import.

File: package.json [hunk 2]
```diff
@@ -89,6 +89,7 @@
     "react-mapbox-gl": "^5.1.0",
     "react-mapbox-gl-draw": "^2.0.4",
     "react-player": "^2.8.2",
+    "react-use": "^17.2.1",
```
Comment (norbertschuler) [inline package.json:92]: This is another (big) package - is it necessary? https://bundlephobia.com/result?p=react-use@17.2.1

Comment (rishsingh07) [reply package.json:92]: I'll look for some solution to it.

Comment (norbertschuler) [reply package.json:92]: Generally it is no problem if we add a package if it is the best solution for the issue and everything else is too difficult. We just should have a short check before, if the problem can not be solved easily without it.

**Comment by sagararyal** [comment]
why do we need this feature?

**Pull request closed without merging**